src/common/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ target_link_libraries(common "${CMAKE_CURRENT_LIST_DIR}/thirdparty/hiredis/libhi
 
 function(define_test test_name library)
   add_executable(${test_name} test/${test_name}.c ${ARGN})
-  add_dependencies(${test_name} hiredis flatcc)
+  add_dependencies(${test_name} hiredis flatcc flatbuffers_ep)
   target_link_libraries(${test_name} common ${FLATBUFFERS_STATIC_LIB} ${library})
   target_compile_options(${test_name} PUBLIC "-DPLASMA_TEST -DPHOTON_TEST -DCOMMON_TEST -DRAY_COMMON_LOG_LEVEL=4 -DRAY_TIMEOUT=50")
 endfunction()
```

Comment:
These .gitkeeps are not used any more. Not sure it should be part of this PR but they should be removed...